study.md

```diff
@@ -44,21 +44,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+View-state is the method that the page framework uses to preserve page and controlled values.  There are multiple different views that exist on the same page meaning it doesn't need to refresh.
 ```
```

Comment:
Good! The routers are what manage view-state by responding to user inputs, and directing them to the appropriate view.